core/frontend/src/components/parameter-editor/ParameterEditor.vue

```diff
@@ -0,0 +1,313 @@
+<template>
+  <v-card
+    width="700"
+    elevation="0"
+    class="mx-auto my-6"
+  >
+    <v-data-table
+      v-if="finished_loading"
+      :headers="[{ text: 'Name', value: 'name' },
+                 { text: 'Description', value: 'description' },
+                 { text: 'Value', value: 'value' }]"
+      :items="params"
+      item-key="name"
+      class="elevation-1"
+      :search="search"
+      :custom-filter="filterOnlyCapsText"
+      :sort-by="'name'"
+      @click:row="editParam"
+    >
+      <template #top>
+        <v-text-field
+          v-model="search"
+          label="Search"
+          class="mx-4"
+        />
+      </template>
+      <template #item.value="{ item }">
+        {{ printParam(item) }}
+      </template>
+    </v-data-table>
+    <spinning-logo
+      v-else
+      :subtitle="`Loading parameters: ${Object.keys(params).length ?? 'loading'}`"
```

Comment:
you can call length directly
```suggestion
      :subtitle="`Loading parameters: ${params.length ?? 'loading'}`"
```